dll/win32/shell32/dialogs/recycler_prop.cpp

```diff
@@ -185,13 +191,19 @@ static BOOL StoreDriveSettings(HWND hwndDlg)
     HWND hDlgCtrl = GetDlgItem(hwndDlg, 14000);
     LVITEMW li;
     PDRIVE_ITEM_CONTEXT pItem;
-    HKEY hKey, hSubKey;
+    HKEY hKey, hSubKey, hExploreKey;
     WCHAR szSerial[20];
     DWORD dwSize;
 
     if (RegCreateKeyExW(HKEY_CURRENT_USER, L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\BitBucket\\Volume", 0, NULL, 0, KEY_WRITE, NULL, &hKey, NULL) != ERROR_SUCCESS)
         return FALSE;
 
+    if (RegCreateKeyExW(HKEY_CURRENT_USER, L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explore", 0, NULL, 0, KEY_WRITE, NULL, &hExploreKey, NULL) != ERROR_SUCCESS)
```

Comment:
Since this is only needed for storing the "ConfirmFileDelete" value (done at the very end of the function), is it necessary to create/open it now and fail everything if it's not possible create it?

Reply:
well, it's a fail oviously, but maybe this could be done at the end of the function

Reply:
That's so that the dialog could be displayed and the other values that don't need that registry key be viewed, set and saved.

Reply:
Ok done